code/addons/essentials/src/index.ts

```diff
@@ -38,15 +39,16 @@ export function addons(options: PresetOptions) {
 
   const main = requireMain(options.configDir);
   return [
-    'docs',
-    'controls',
     'actions',
```

Comment:
I'm not 100% sure here, but I believe this order was this way for a reason. Addon docs should be loaded before others, so controls and actions work as expected (as they need docgen for certain features)

Reply:
Shoot! I'll revert this and add a comment